review-service/index.js

```diff
@@ -17,7 +17,7 @@ const router = express.Router()
 
 router.get('/', getUserScores)
 router.post('/', createScores)
-router.delete('/', deleteUserScoresFromCache)
+router.delete('/cache', deleteUserScoresFromCache)
```

Comment:
I just thought it'll be clearer to a client what exactly they're deleting/clearing 😬